.github/workflows/build.yml

```diff
@@ -0,0 +1,79 @@
+name: Build
+
+on: push
+
+jobs:
+  patch-src:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: GitHub Environment Variables Action
+        uses: FranzDiebold/github-env-vars-action@v1.2.1
```

Comment:
Do we need to `echo` the in variables here? Like in the example: https://github.com/FranzDiebold/github-env-vars-action/blob/primary/.github/workflows/demo.yml

Reply:
No, that is to show how they are used. They are used later on, for the naming of the artifacts